### PR TITLE
add dockerfile

### PR DESCRIPTION
**Pull request by wxchan**
I already test it. pls give another test before merge it.

### CODE REVIEW

File: docker/dockerfile-python [hunk 1]
```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:16.04
+MAINTAINER wxchan
```
Comment (henry0312) [inline docker/dockerfile-python]: `MAINTAINER` is deprecated from v1.13.0.
https://github.com/docker/docker/releases/tag/v1.13.0

File: docker/dockerfile-python [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+RUN apt-get update && \
+    apt-get install -y cmake build-essential gcc g++ git wget && \
+
+# open-mpi
+    cd /usr/local/src && mkdir openmpi && cd openmpi && \
```
Comment (henry0312) [inline docker/dockerfile-python]: I don't think it's necessary to insert indent to each line :p

File: docker/dockerfile-python (hunk 1 above)
Comment (wxchan) [inline docker/dockerfile-python]: @henry0312 so should I just remove it or use `LABEL maintainer`?

File: docker/dockerfile-python (hunk 2 above)
Comment (wxchan) [inline docker/dockerfile-python]: it looks better, I think

File: docker/dockerfile-python (hunk 1 above)
Comment (henry0312) [inline docker/dockerfile-python]: Yes, just removing is ok.

File: docker/dockerfile-python (hunk 2 above)
Comment (henry0312) [inline docker/dockerfile-python]: yeah, but this doesn't seem common....

**Pull request merged**